elfi/methods/bo/acquisition.py

```diff
@@ -161,7 +161,7 @@ def _beta(self, t):
         d = self.model.input_dim
         return 2*np.log(t**(2*d + 2) * np.pi**2 / (3*self.delta))
 
-    def evaluate(self, x, t=None):
+    def evaluate(self, x, t=0):
```

Comment:
What would be the reason for this change? I think it's better to take the default value away completely than to make a default that may hide the fact that no valid value was passed.